src/main/ts/components/Chart.tsx

```diff
@@ -114,59 +182,68 @@ export default function Chart() {
-            yScale={yScale}
-            chartEndDate={chartEndDate}
-          />
-          <ChartLine
-            data={caycProjectionData}
-            xScale={xScale}
-            yScale={yScale}
-            chartEndDate={chartEndDate}
-            projection={true}
-          />
-        </Graph>
-        <GraphAnnotation>
-          <img aria-hidden={true} alt="arrow" src={ARROW} />
-          <IssuesDeltaText>
-            <Paragraph>{translate('cayc.chart.nudge')}</Paragraph>
-            <Paragraph>
-              <FormattedMessage
-                id="cayc.chart.fewer_issues"
-                defaultMessage={translate('cayc.chart.fewer_issues')}
-                values={{
-                  count: <strong>{issuesDelta}</strong>,
-                }}
+        {hasData && !isLoading && (
+          <>
+            <Graph height={GRAPH_HEIGHT} width={GRAPH_WIDTH + CHART_SIDEBAR_WIDTH}>
+              <ChartVerticalLabel
+                label={format(chartStartingDate, GRAPH_VERTICAL_MARKER_DATE_FORMAT)}
+                labelYOffsetLevel={3}
+                x={0}
               />
-            </Paragraph>
-          </IssuesDeltaText>
-        </GraphAnnotation>
+              <ChartVerticalMarker
+                xScale={xScale}
+                date={caycStartingDate}
+                dash
+                label={format(caycStartingDate, GRAPH_VERTICAL_MARKER_DATE_FORMAT)}
+                labelYOffsetLevel={2}
+              />
+              {cumulativeData.length > 0 && (
+                <ChartVerticalMarker
+                  xScale={xScale}
+                  date={chartEndDate}
+                  label={translate('cayc.chart.now')}
+                  labelYOffsetLevel={1}
+                />
+              )}
+              <ChartLine
+                data={cumulativeData}
+                xScale={xScale}
+                yScale={yScale}
+                chartEndDate={chartEndDate}
+              />
+              <ChartLine
+                data={caycProjectionData}
+                xScale={xScale}
+                yScale={yScale}
+                chartEndDate={chartEndDate}
+                projection
+              />
+            </Graph>
+            <GraphAnnotation>
+              <img aria-hidden alt="arrow" src={ARROW} />
+              <IssuesDeltaText>
+                <Paragraph>{translate('cayc.chart.nudge')}</Paragraph>
+                <Paragraph>
+                  <FormattedMessage
+                    id="cayc.chart.fewer_issues"
+                    defaultMessage={translate('cayc.chart.fewer_issues')}
+                    values={{
+                      count: <strong>{issuesDelta()}</strong>,
+                    }}
+                  />
+                </Paragraph>
+              </IssuesDeltaText>
+            </GraphAnnotation>
+          </>
+        )}
+        {!hasData && !isLoading && (
+          <div>
+            <p>{translate('cayc.no_data')}</p>
+          </div>
```

Comment:
I don't think you need the `div`, do you?
```suggestion
    <p>{translate('cayc.no_data')}</p>
```